### PR TITLE
feat(Data Mapper): Merge with Designer Vs-code Extension

### PR DESCRIPTION
**Pull request by Elaina-Lee**
- **Please check if the PR fulfills these requirements**

* [X] The commit message follows our guidelines

- **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
- Merge Vs-code data mapper extension with existing designer vs-code extension
- **What is the current behavior?** (You can also link to an open issue here)
- Data mapper has its own extension
- **What is the new behavior (if this is a feature change)?**
- Designer vs-code extension contains data mapper
- **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

- **Please Include Screenshots or Videos of the intended change**:
0

AB#25597025

### CODE REVIEW

**Comment by ccastrotrejo** [comment]
There are some Unchanged files with check annotations  from the Github actions  in`libs/data-mapper/src/lib/ui/ReactFlowWrapper.tsx` Could you check those?

**Comment by Elaina-Lee** [comment]
> There are some Unchanged files with check annotations from the Github actions in`libs/data-mapper/src/lib/ui/ReactFlowWrapper.tsx` Could you check those?

Checked and fixed. The warning seems to be gone now!

File: apps/vs-code-designer/src/app/commands/dataMapper/extensionConfig.ts
Comment (ccastrotrejo) [inline apps/vs-code-designer/src/app/commands/dataMapper/extensionConfig.ts:1]: NIT: Some of this values are already in `apps/vs-code-designer/src/constants.ts` just check which ones are duplicated

**Review by ccastrotrejo — APPROVED** [review]

File: apps/vs-code-designer/src/README.md [hunk 1]
```diff
@@ -4,6 +4,10 @@ In Visual Studio Code, you can use the Azure Logic Apps (Standard) extension to
 
 > Sign up today for your free Azure account and receive 12 months of free popular services, $200 free credit and 25+ always free services 👉 [Start Free](https://azure.microsoft.com/free/open-source).
 
+# Azure Logic Apps - Data Mapper extension for Visual Studio Code
```
[the remark this answers is not in the document]
Comment (Elaina-Lee) [reply apps/vs-code-designer/src/README.md]: removed

File: apps/vs-code-designer/src/app/commands/dataMapper/extensionConfig.ts
Comment (Elaina-Lee) [reply apps/vs-code-designer/src/app/commands/dataMapper/extensionConfig.ts:1]: Moved all the duplicates and also the constants that seem to be shared.

**Review by ccastrotrejo — APPROVED** [review]

**Pull request merged**